translations/en-us.yaml

```diff
@@ -57,3 +58,4 @@ oss-components:
       subtitle: Failed to copy to your clipboard. Please try again.
   access-panel:
     search_placeholder: Search...
+    
```

Comment:
missing newline ? or extra newline ? either way github is saying something is wrong